crates/tauri/src/tray/mod.rs

```diff
@@ -603,6 +603,18 @@ impl<R: Runtime> TrayIcon<R> {
       })
     })
   }
+
+  /// Get the inner tray icon.
+  ///
+  /// Note that `tray-icon` crate may be updated in minor releases of Tauri.
+  /// Therefore, it’s recommended to pin Tauri to at least a minor version when you’re using `with_inner_tray_icon`.
+  pub fn with_inner_tray_icon<F, T>(&self, f: F) -> crate::Result<T>
+  where
+    F: FnOnce(tray_icon::TrayIcon) -> T + Send + 'static,
```

Comment:
I think we can avoid the clone here right? `&tray_icon::TrayIcon` should be able to do everything needed, and if people want, they could in rare cases clone it themselves

Reply:
@Legend-Master fixed 🫡